app/views/ads/show.html.erb

```diff
@@ -50,5 +50,13 @@
   <%= @ad.description %>
 </p>
 
+<div style= 'width: 400px'>
+  <strong>IMAGE:</strong>
+  <% @ad.images.each do |i| %>
+    <%= link_to image_tag(i, style: 'width: 100%'), ads_path %>
```

Comment:
we don't style in html tags, instead create a class `width-100` and use css for this